packages/m/mdb-ui-kit.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "mdb-ui-kit",
+  "filename": "js/mdb.min.js",
+  "description": "Material Design for Bootstrap (MDB) is a powerful Material Design UI KIT for most popular HTML, CSS, and JS framework - Bootstrap.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mdbootstrap/mdb-ui-kit"
+  },
+  "keywords": [
+    "material",
+    "design",
+    "bootstrap",
+    "mdbootstrap",
+    "mdb",
+    "framework",
+    "css",
+    "mobile-first",
+    "responsive"
+  ],
+  "author": "MDBootstrap",
+  "license": "MIT",
+  "homepage": "https://mdbootstrap.com/docs/standard/",
+  "autoupdate": {
+    "source": "npm",
+    "target": "mdb-ui-kit",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/mdb.min.js",
+          "css/mdb.min.css"
+        ]
+      }
+    ]
```

Comment:
```suggestion
    "fileMap": [
      {
        "basePath": "js",
        "files": [
          "mdb*"
        ]
      },
      {
        "basePath": "css",
        "files": [
          "mdb*"
        ]
      },
      {
        "basePath": "src",
        "files": [
          "scss/**/*.scss"
        ]
      }
    ]
```

Let's do a few things here unless you disagree with this:

- Pull the js & css files up, out of their subdirectories, into the root of the version on cdnjs
- Include the maps for the js & css files, as we support that extension
- Include the scss files (in a `scss` subdirectory, pulled out of `src`) as we support them and folks using Codepen/Repl.it/Libman might want them

[Rough glob preview](https://www.digitalocean.com/community/tools/glob?c=Y29tbWVudHM9dHJ1ZSZnbG9iPSU3QmNzcyUyRm1kYiUyQSUyQ2rNDnNyYyUyRnPGJMUTQcYJLsUUN0QmbWF0Y2hlcz1mYWxzZSZ0ZXN0cz3FI0YlMjBJbXBvcnRlZCUyMGZyb20lMjBtZGItdWkta2l0JTIwTlBNJTIwcGFja2FnyDzGa21kYi5taW4uY3NzxxhpbmRleC5odG1syBFtZ8YpLWZhdmljb24uaWNvxxzmALflAMtib290c3RyYXAlMkZkaXN0JTJGYWxlcnQuasha1y3GF9osZnJlZcQhbmltYXRl30%2FJfGJ1dHRvbt8uyH3fLc9bY2Fyb3VzZWzfMMhd3y%2FSX29sbGFwc%2F8A6spf3y%2FQX2TkAh9GZGF0Yd8yyGHfMeUCw98l9QCIcm9wZG93%2FwF26gCG3C%2FnAiPfU%2FcBC2V2ZW50LWhhbmRsZXLfO%2B4BFN867wEd3y7JLnV0aWzkANRvY3VzVHJhcNUr8gCTxzfmA7%2FfXt8nyid0b3VjaN0v5wF7aW5wdfYDv81Qa2V5Y29kZXPVKvkBom1hbmlwdWxhdG%2F%2FAaDvAaDfOO0Bnt8sxCwu5ADsyh3JF%2BQFQP8Al%2BgA0G1vZGH%2FBAnpAMTfLM5Z5gDFcG9seWZpbN827wEm3TXsASPfXvEAlHBvcG92%2FwNT6gCN3C7nAmVyaXBwbP8E6fADEHNhbml0aXrXWe0Cnd4r8wDhc2Nyb2xsc3B53zHoAJPfMNJh5gF4c2VsZWN0b3ItZW5naW7%2FAQDvAd3fPPEB5N8w%2BgQic3dpcNYv8wDYdGFi3yvoAMbfKs1Vb2Fz%2FwhuylffLM9Zb2x0af8FiMtb2i71ATrFCFV0afYDWOcC%2B3RyZWV2aWV3yiTnCc4uanNvbv8Ax%2B4Jji5tYXDfMe8JQ98y9Qka3zT1CO%2FfNPcIxN829Qhy3zT%2FCCPfP%2F8GwMs98QuEyxzvBl%2Ffc%2FIGY98x%2BwY73zr0BeHfM%2FYFM981%2FwUH30HNQeYERt8v8gRK3zH0BCLLM1JFQURNRS5tZMcQTGljZW5zZS5wZGbNVecN9cxXX%2BYDrMQa3ytGbWl4aW7EFd805wRI3ybHJt9V0lXmDZwtZXh0ZW5kZWTfMckx2SjsAOLkBS9pdGllxBhfYXBp3zXGNV9iYWRn2mDoAIjeJvYBaGJvcmRlci1yYWRpdXPfPNI8eC1zaGFkb3ffOcY5X2JyZWFkY3J1bWLZMOkAy98r%2BgCUcmVha3BvaW50%2FwDOxzpf5gYYLWdyb3Vw2TLpAJffLdhf%2FwCM8gDG3zbuAJDfKN9Z1TFvZHVs5gKk3zLuAO5jYXL%2FAy%2FfJfcBEGNhcmX%2FBELrAIPmB7bfLsUuaGVscGVyxmNsZWFyZml43zjxAJrfN803X2Nsb3P%2FA6LzAJpvbG9yZWQtbGlua%2F8Busgu3yftAMHeMPcA9m9udGFpbmVy3zjGOMov32jRaGRlcHJlY%2F8FXMpo6QkU2C7oASXfKfkBi2VtYv8GEMQmZmxhZ98lRl9mb290%2FwFA6ACBZm9ybcV%2FxAktY2hlY2vfX98z32vKOG9udHJv%2BgNL1m3fNdpvZmls%2Fwbl3zLfMsQyb3V0bOQJcNg1%2BgCecmFu%2FweE3zPfa8s45god32zfNN9tyjl0ZXjfa98y2mnFL%2F8D9PED9N80zV91bmN0af8G0OYDbt8q%2BACNZ%2BQJGeQY%2Ft84xjhfZ3Jp%2Fwbb02LfM8ld5wSI3y3QYGlt5A9Y3zTGYcUr2mDoAUzfJ%2FUCauUVUv8JOPACbN802m1sYWJlbP8AyN8v12NfbGlz%2FwCT9wGH3zneOf8Ancc0X8Y12SzoAcDfJ85T5Q%2BA31LfJs1RbmF230%2FeJNBNYmH6B0TLUNwnzVNwYWdpbmHkBCvfMPABqN857ACQ3yvQZF%2FoEJrfWN8o%2BQjVcG9zaf8Ajco4X3Byb2dyZXP%2FAmffKdJXcmXED99V3yf4AZtyZXNldP8Gttg5aXr%2FBT3HNXZlbmRvcsU1Zv8BHMQk5hf83yfrAIneMO4BSv8BSMRa3iX2AQ1zY3JlZW4t5A%2BZ%2FwJk8wJkxj3GPP8BH8Yu5xNY3ypf5hB%2F2lLtARpzcGlu%2FwyF9gC9dHJl5CCv5g2D2T5jdXN05RPcX3N0eWz%2FByLyAWB0YWJsZS12YXJpYf8IcMk9xjTfLNJpZXh0LXRydW7%2FDVn1AQzMPf8ENMk%2FX%2BUUg%2F8B998n0lPmFKXfVN8o%2BQEjcmFu%2FwUVzDnLMP8Av8kxeXD%2FA%2F%2FfJc5P6RRW3y%2FwAOjfOPgJYXZhbGlk%2Fwbs7wll3zPQa1%2FlAwj9At3qA2%2FfLOsBUt8q7wCFyQwt%2Fwul1DP%2FBqrWNf8BkNg42S7kGXJjb3L6AkbpIAnLIecX63R4dA%3D%3D)